packages/launcher/test/unit/windows_spec.ts

```diff
@@ -87,6 +89,46 @@ describe('windows browser detection', () => {
     snapshot(await detect(firefoxes))
   })
 
+  it('works with :browserName format in Windows', () => {
+    sinon.stub(os, 'platform').returns('win32')
+    stubBrowser(`${HOMEDIR}/foo/bar/browser.exe`, 'foo-browser v100')
+
+    return detectByPath(`${HOMEDIR}/foo/bar/browser.exe:foo-browser`, goalBrowsers as Browser[]).then((browser) => {
+      expect(browser).to.deep.equal(
+        Object.assign({}, goalBrowsers.find((gb) => {
+          return gb.name === 'foo-browser'
+        }), {
+          displayName: 'Custom Foo Browser',
+          info: `Loaded from ${HOMEDIR}/foo/bar/browser.exe`,
+          custom: true,
+          version: '100',
+          majorVersion: 100,
+          path: `${HOMEDIR}/foo/bar/browser.exe`,
+        }),
+      )
+    })
+  })
+
+  it('identifies browser if name in path', async () => {
+    sinon.stub(os, 'platform').returns('win32')
+    stubBrowser(`${HOMEDIR}/foo/bar/chrome.exe`, 'Google Chrome 100')
```

Comment:
Same as above - see `stubBrowser` impl

```suggestion
    stubBrowser(`${HOMEDIR}/foo/bar/chrome.exe`, '100')
```